examples/srd_v1_3x2.yaml

```diff
@@ -0,0 +1,241 @@
+generate:
```

Comment:
The use of yaml anchors in this file is clever for sure. Maybe instead though we could have an option where if people do not specify the second config, it defaults to the first? 

I am so so on this feature, but I am sensitive to the amount of YAML ppl will have to write here.

Reply:
not sure what you mean ;)